### ISSUE TITLE
Allow redefining predefined sorting criteria.

### ISSUE THREAD

**Issue by erling6232** [erling6232/imagedata#32]
E.g. allow Series(path, 'b', opts = {'b': 'InstanceNumber'})

Predefine sorting criteria in opts, and allow these to be modified by the user.

Modify dicomplugin._get_tag()

What should go into opts to predefine 'time' and 'b'?

**Comment by erling6232** [comment]
This issue was intended to have a limited approach: At present, time resolved acquisitons are sorted on the DICOM attribute AcquisitionTime. If this does not work for a data set, you can invent another search criteria and name this like "t". This is documented in the manual.
However, at present, you cannot redefine the predefined "time" sort field to something else, like InstanceNumber. It would be good to redefine the "time" sorting, instead of inventing another "t" sorting.

**Comment by HaukeBartsch** [comment]
Yes, would be good to have a more flexible approach (for example the one outlined above). Some acquisitions (seen this on GE and Siemens) randomize the acquisition of individual slices, apparently this improves image quality for some sequences. Because of that AcquisitionTime is probably not a good default sorting. Might be better to start with SliceLocation.
